modules/affile/affile-dest.c

```diff
@@ -97,7 +97,7 @@ struct _AFFileDestWriter
   time_t last_msg_stamp;
   time_t last_open_stamp;
   time_t time_reopen;
-  struct iv_timer reap_timer;
+  struct iv_timer retry_timer;
```

Comment:
You're right about that but the duplication of timers when they can't be active at the same time is a noise in the code. Your idea about `log_pipe_notify` would be great, though. Moving the responsibility of trying to reopen the destination to the logwriter can solve the problem and a much nicer solution. Next to that it can be used in many different destinations where the same problem exists.